index.dd

```diff
@@ -76,6 +76,22 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE
+----
+// Compile-Time Sort in D
+void main()
+{
+    import std.algorithm, std.stdio;
+    writeln("Starting program.");
+    // Constant declaration
```

Comment:
I think we can't really go with EOL comments because they might not be directly visible on mobile devices,  but sure a new line doesn't hurt :)